app/survey.js

```diff
@@ -120,35 +130,56 @@ export default class Survey extends Component {
     )
   };
 
+  submitPressed() {
+   console.log(this.state.answers);
+   AsyncStorage.getItem('username').then((value)=> {
+    fetch("http://107.170.234.65:8000/Glia/get-user-id/?username=" + encodeURI(value) + "&phq9=" + encodeURI(this.state.answers.toString()),{
```

Comment:
this makes sense for now, because I saw that you secured the api - but hopefully we can eventually upload this ph q9 data to a hippa compliant service #13